static/lang/re-en.json

```diff
@@ -293,7 +293,8 @@
                 },
                 "Moderate": {
                     "success": "<strong>Effect</strong> The bomb also deals @Damage[(4[splash])[piercing]]{4 piercing splash damage} and the splash creates crystals that are @UUID[Compendium.pf2e.equipment-srd.Item.7fSnvJ2xoSfa6JXD]{Caltrops} on level surfaces and handholds on vertical surfaces, granting +1 item bonus to the next Athletics check to Climb."
-                }
+                },
```

Comment:
Can the bleed damage to added directly to the bomb damage? That would then double on a critical which I believe is correct.

We could then remove all of the inline damage rolls.